2_ThreeSum.cpp

```diff
@@ -114,9 +114,9 @@ int main()
 	vector<vector<int> > result = threeSum(inp);
   	for (int i = 0; i < result.size(); ++i) 
 	{
-    	for (int j = 0; j < 3; ++j)

```

Comment:
I haven't done C++ in a while, but is freopen the common way to read files?  I seem to recall using file streams for these operations

Reply:
File streams could also be used. But I switched mode to stdin after opening the file as I'm scanning integer by integer, hence used freopen. 'If a new filename is specified, the function first attempts to close any file already associated with stream (third parameter) and disassociates it. Then, independently of whether that stream was successfuly closed or not, freopen opens the file specified by filename and associates it with the stream just as fopen would do using the specified mode.'